CHANGELOG.md

```diff
@@ -107,14 +107,14 @@ pub enum PollError {
 
 > [!WARNING]
 > As part of this change, WebGL's default behavior has changed. Previously `device.poll(Wait)` appeared as though it functioned correctly. This was a quirk caused by the bug that these PRs fixed. Now it will always return `Timeout` if the submission has not already completed. As many people rely on this behavior on WebGL, there is a new options in `BackendOptions`. If you want the old behavior, set the following on instance creation:
-> 
+>
 > ```rust
 > instance_desc.backend_options.gl.fence_behavior = wgpu::GlFenceBehavior::AutoFinish;
 > ```
-> 
+>
 > You will lose the ability to know exactly when a submission has completed, but `device.poll(Wait)` will behave the same as it does on native.
 
-By @cwfitzgerald in [#6942](https://github.com/gfx-rs/wgpu/pull/6942).  
+By @cwfitzgerald in [#6942](https://github.com/gfx-rs/wgpu/pull/6942).
```

Comment:
We should either:

- Remove these changes
- Split into separate PR
- Split into separate commits, and squash the other commits, so we can rebase-and-merge.

Reply:
This is just formatting the changelog to remove whitespace (my editor auto-performed it). Do you really want a separate commit/PR? If so I'd rather just revert that part.

Reply:
Ehhhhhhhhh, just merged it anyway, probably not a big deal